src/openclpow.py

```diff
@@ -55,24 +60,30 @@ def initCL():
         else:
             logger.info("No OpenCL GPUs found")
             del enabledGpus[:]
-    except Exception as e:
+    except Exception:
         logger.error("OpenCL fail: ", exc_info=True)
         del enabledGpus[:]
 
+
 def openclAvailable():
-    return (len(gpus) > 0)
+    """Are there any OpenCL GPUs available?"""
+    return bool(gpus)
+
 
 def openclEnabled():
-    return (len(enabledGpus) > 0)
+    """Is OpenCL enabled (and available)?"""
+    return bool(enabledGpus)
 
-def do_opencl_pow(hash, target):
+
+def do_opencl_pow(hash_, target_):
```

Comment:
what's wrong with target?

Reply:
target is already used in line no.117. So because of that pylint shows Redefining name 'target' from an outer scope.

Reply:
so replace it in line 117 - in script, not in function definition

Reply:
it's clearly NOT an outer scope